CONTRIBUTING.md

```diff
@@ -38,3 +38,13 @@ The CI will run several checks on the new code pushed to the Gym repository. The
 2. install the Git hooks by running `pre-commit install`.
 
 Once those two steps are done, the Git hooks will be run automatically at every new commit. The Git hooks can also be run manually with `pre-commit run --all-files`, and if needed they can be skipped (not recommended) with `git commit --no-verify`. **Note:** you may have to run `pre-commit run --all-files` manually a couple of times to make it pass when you commit, as each formatting tool will first format the code and fail the first time but should pass the second time.
+
+## Docstrings
+Pydocstyle has been added to the pre-commit process such that all new functions follow the (google docstring style)[https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html].
+All new functions require either a short docstring, a single line explaining the purpose of a function
+or a multiline docstring that documents each argument and the return type (if there is one) of the function.
+In addition, new file and class require top docstrings that should outline the purpose of the file/class.
+For classes, code block examples can be provided in the top docstring and not the constructor arguments.
+
+To check your docstrings are correct, run `pre-commit run --al-files` or `pydocstyle --source --explain --convention=google`.
+If all docstrings that fail, the source and reason for the failure is provided. 
```

Comment:
I think if should be something like for, right?

Reply:
As VSCode and the Sphinx google code example only uses `Note:`, Im going to keep it as 'Note:' until we move to the website and it becomes an issue